server/src/schema.js

```diff
@@ -56,6 +72,7 @@ const typeDefs = gql`
         tags: [Tag]!
         participants: [User]!
     }
+
     # Every event can have multiple tags.
     # Tags are predefined by ours(developers)
```

Comment:
Each line can have up to 80 characters. First line looks like it can hold more words. Can you double check?

Reply:
If I combine those two lines, It is 81 characters. So I separate them into two lines.